.github/workflows/release.yaml

```diff
@@ -351,6 +353,9 @@ jobs:
           tag: ${{ env.VERSION }}
           overwrite: true
 
+{%- endraw -%}
+{%- if itch_username != "" -%}
+{%- raw -%}
```

Comment:
This shouldn't be necessary. The release workflow already checks if the `ITCH_TARGET` env variable is provided, and if not, doesn't upload to itch.io.

In fact this section should be left in, to allow the user to enable itch.io later on.

Reply:
(redundant with Jan's comment)